### PR TITLE
[LOOP-1847] Prompt for manual BG entry

### PR DESCRIPTION
**Pull request by nhamming**
https://tidepool.atlassian.net/browse/LOOP-1847

present manual BG prompt

### CODE REVIEW

File: Loop/Views/BolusEntryView.swift [hunk 1]
```diff
@@ -42,10 +42,9 @@ struct BolusEntryView: View, HorizontalSizeClassOverride {
                 .padding(.top, self.shouldAutoScroll(basedOn: geometry) ? -200 : -28)
                 .listStyle(GroupedListStyle())
                 .environment(\.horizontalSizeClass, self.horizontalOverride)
-                
                 self.actionArea
-                    .frame(height: self.isKeyboardVisible ? 0 : nil)
-                    .opacity(self.isKeyboardVisible ? 0 : 1)
+                    .frame(height: self.viewModel.isManualGlucoseEntryEnabled || self.isKeyboardVisible ? 0 : nil)
```
Comment (rickpasetto) [inline Loop/Views/BolusEntryView.swift]: Hm...this looks suspicious to me.   What does this change achieve?

**Review by rickpasetto — COMMENTED** [review]
I've added a question that I'd like answered, please, before proceeding with review.  Thanks!

File: Loop/Views/BolusEntryView.swift (hunk 1 above)
Comment (nhamming) [reply Loop/Views/BolusEntryView.swift]: Oh... good catch. I was exploring alternatives with the animation and forgot to remove this.

**Review by rickpasetto — APPROVED** [review]
LGTM 🦖

**Review by darinkrauss — APPROVED** [review]
LGTM! ⛵

**Pull request merged**